mod_reforged/hooks/entity/tactical/humans/barbarian_champion.nut

```diff
@@ -62,27 +62,12 @@
 
 	q.assignRandomEquipment = @(__original) function()
 	{
-	    __original();
-	    local weapon = this.getMainhandItem();
-	    if (weapon != null)
-	    {
-	    	if (weapon.isWeaponType(::Const.Items.WeaponType.Axe))
-		    {
-		    	this.m.Skills.add(::new("scripts/skills/perks/perk_coup_de_grace"));
-		    	this.m.Skills.add(::new("scripts/skills/perks/perk_sundering_strikes"));
-		    	this.m.Skills.add(::new("scripts/skills/perks/perk_rf_cull"));
-		    }
-		    else if (weapon.isWeaponType(::Const.Items.WeaponType.Cleaver))
-		    {
-		    	this.m.Skills.add(::new("scripts/skills/perks/perk_coup_de_grace"));
-		    	this.m.Skills.add(::new("scripts/skills/perks/perk_mastery_cleaver"));
-		    	this.m.Skills.add(::new("scripts/skills/perks/perk_rf_mauler"));
-		    }
-		    else
-		    {
-		    	::Reforged.Skills.addPerkGroupOfEquippedWeapon(this);
-		    }
-	    }
+		__original();
+		local weapon = this.getMainhandItem();
+		if (weapon != null)
+		{
+			::Reforged.Skills.addPerkGroupOfEquippedWeapon(this);
+		}
```

Comment:
This entire thing can be replaced with just `::Reforged.Skills.addPerkGroupOfEquippedWeapon(this);` i.e. no need to have the weapon be accessed locally as a variable if you aren't going to use it.